release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,26 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 17 September 2024 - Enhancement - Admin - Improved message when no usage data is found [ID 40796]
+
+Improved the message when no usage data is available yet for a given organization.
+
+#### 17 September 2024 - New feature - Admin - STaaS region registration and configuration tile [ID 40786]
+
+It is now possible to register your DataMiner system for STaaS through the Admin app.
+
+#### 17 September 2024 - Fix - Admin - Incorrect notation usage number [ID 40781]
+
+The notation of the number 400,000 has been changed from 0,4M to 400K.
+
+#### 17 September 2024 - Enhancement - Catalog - Catalog registration now only possible via API [ID 40772]
+
+It is now no longer possible to register Catalog items using the Catalog app. Registration should be done using the Catalog API instead. For detailed information, see [docs](https://docs.dataminer.services/develop/CICD/Skyline%20Communications/Publish%20to%20Catalog/Publish_To_Catalog.html).
```

Comment:
It seems odd to link to that "Publishing artifacts to the DataMiner Catalog" page when there's a more suitable page that was added recently. I'd suggest this link instead:
```suggestion
It is now no longer possible to register Catalog items using the Catalog app. Registration should be done using the Catalog API instead. For detailed information, see [Registering a Catalog item](xref:Register_Catalog_Item).
```

However, we may also need to update the page you originally linked to so that it too contains a link to the page about registering a Catalog item.

Reply:
Will be updated once the Catalog Api is integrated in their pipelines